.github/workflows/main.yml

```diff
@@ -14,14 +14,16 @@ jobs:
         os: [ubuntu-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
-      - name: Library Build & Test
+      - name: Install FFmpeg - linux
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get install ffmpeg -y
+      - name: Build
         run: |
           dotnet build -c Release
-          dotnet test -c Release
-      - name: Build Redist windows x64
```

Comment:
This looks like another bad auto-merge, it's covered in the previous step already